openmetadata-docs/content/v1.3.x/main-concepts/metadata-standard/schemas/tests/column/columnValueMaxToBeBetween.md

```diff
@@ -1,16 +0,0 @@
----
-title: columnValueMaxToBeBetween
-slug: /main-concepts/metadata-standard/schemas/tests/column/columnvaluemaxtobebetween
----
-
-# columnValueMaxToBeBetween
-
-*This schema defines the test ColumnValueMaxToBeBetween. Test the maximum value in a col is within a range.*
-
-## Properties
-
-- **`minValueForMaxInCol`** *(integer)*: Expected maximum value in the column to be greater or equal than.
-- **`maxValueForMaxInCol`** *(integer)*: Expected maximum value in the column to be lower or equal than.
-
-
-Documentation file automatically generated at 2022-07-14 10:51:34.749986.
```

Comment:
where did we move these docs?

Reply:
this diff is not clean.. in any case they are here:
https://github.com/open-metadata/OpenMetadata/blob/a4c516d2c70c69ae8cd27096da077a5b62d4a53f/openmetadata-docs/content/v1.3.x/connectors/ingestion/workflows/data-quality/tests.md#L4